src/main/kotlin/no/nav/tilleggsstonader/soknad/soknad/barnetilsyn/BarnetilsynMapper.kt

```diff
@@ -45,6 +46,14 @@ class BarnetilsynMapper {
             navn = TekstFelt(labelNavn(språkkode), barn.visningsnavn),
             ident = TekstFelt(labelFødselsnummer(språkkode), it.ident),
             type = it.type,
+            utgifter =
+                it.utgifter?.harUtgifterTilPass?.let { it1 ->
+                    Utgifter(
+                        harUtgifterTilPass = it1,
+                        fom = it.utgifter.fom,
+                        tom = it.utgifter.tom,
+                    )
+                },
```

Comment:
Her kan man ev sette let etter utgifter for å gjøre det tydligere at det er utgifter som man sjekker er null eller ikke
```suggestion
                it.utgifter?.let { utgifter ->
                    Utgifter(
                        harUtgifterTilPass = utgifter.harUtgifterTilPass,
                        fom = utgifter.fom,
                        tom = utgifter.tom,
                    )
                },
```